wp-content/plugins/hurumap/data.php

```diff
@@ -17,7 +17,7 @@ function initialize()
 
         function register_post_types()
         {
-            $cap = acf_get_setting('capability');
+            $cap = 'edit_posts';
```

Comment:
Cool, went through the docs again @KobbyMmo:

1. We want to support **Author** role so from [this](https://wordpress.org/support/article/roles-and-capabilities/#capability-vs-role-table) table, we've picked `edit_posts` to represent the common capability between **Author**, **Editor** and **Administrator**,
2. This part of code is still _iffy_ to me: https://github.com/CodeForAfrica/HURUmap-Dashboard/pull/24/files#diff-cf78c359827fa5cb242903d8aba966f3R40-R46

Reply:
@kilemensi 
1. Yes for that, but that is just to show in the menu, it is not to give access to the plugin, that is why there is 2.

2. But going through the [documentation](https://developer.wordpress.org/reference/functions/register_post_type/) I think we are doing it wrong for 2. If we are using the default `'capability_type'    => 'post'`, then we do not need to specify the capabilities in that section, because it will be generated and mapped by default. 

```php
'capabilities' => array(
  'edit_post'          => 'edit_post', 
  'read_post'          => 'read_post', 
  'delete_post'        => 'delete_post', 
  'edit_posts'         => 'edit_post', 
  'edit_others_posts'  => 'edit_others_post', 
  'publish_posts'      => 'publish_post',       
  'read_private_posts' => 'read_private_post', 
  'create_posts'       => 'create_post', 
)

Reply:
That's what I thought too @KobbyMmo

Reply:
But I'm also going through another discussion with Academy team and turns out you need special capabilities to post content with `<script>`, etc. @KobbyMmo ... Can we also just make sure we do/don't need it for HURUmap plugin e.g. how do we post Flourish viz?

Reply:
@kilemensi  who is the Academy team? and what special capabilities are needed to post `<script>`.

Reply:
It's just another team that uses WordPress here at CfA @KobbyMmo ... My point was if you wanted to post content that has `<script>`, for example when you want to embed a chart/iframe, you'd need [`unfiltered_html`](https://wordpress.org/support/article/roles-and-capabilities/#unfiltered_html) capability which only Admininistrator/Super-Admin role has in single-site/multi-site setup respectively.

So, do we use any of this "active" content in our plugin? Especially around Flourish viz.

Reply:
Hello @kilemensi  So I have been investigating the issue.

1. Do they want to add the unfiltered_html to the permissions (That is because even though the editors might be able to add charts, they might not be able to insert it into posts or pages)
2. I haven't been able to fully test out the flourish type because of a 404 error I am getting. But the hurumap type works perfectly.

Reply:
No no @KobbyMmo ... It's just something I wanted us to fully check before lowering required role from **Administrator** to **Author** & looking at the [save](https://github.com/CodeForAfrica/HURUmap-Dashboard/blob/master/wp-content/plugins/hurumap/src/FlourishBlock/Save.js) code, seems that we're saving a plain `div` and converting it to an `iframe` on the [front end](https://github.com/CodeForAfrica/HURUmap-UI/blob/master/packages/core/src/Card/FlourishChart.js) so we don't need `unfiltered_html`

Lets just finalize https://github.com/CodeForAfrica/HURUmap-Dashboard/pull/24/files#diff-cf78c359827fa5cb242903d8aba966f3R40-R46 and I think we'll be done with the PR.